src/modules/fancyzones/editor/FancyZonesEditor/EditorWindow.cs

```diff
@@ -24,7 +24,7 @@ protected void OnSave(object sender, RoutedEventArgs e)
             Logger.LogTrace();
 
             // If new custom Canvas layout is created (i.e. edited Blank layout),
-            // it's type needs to be updated
+            // its type needs to be updated
```

Comment:
Here's one of those cases where looking over all the cases found an instance worth changing...